internal/errors/info.go

```diff
@@ -64,4 +68,8 @@ var errorCodeInfo = map[Code]Info{
 		Message: "multiple records",
 		Kind:    Search,
 	},
+	MaxRetries: {
+		Message: "too many retries",
+		Kind:    Search,
```

Comment:
I am not sure what Kind `ColumnNotFound` should be @mgaffney @jimlambrt?

Reply:
Column not found indicates the database is not in a state it should be which can only be due to internal failure. 
 Maybe this is one of the rare  instances where Internal makes sense.